mautic/mautic
# Config to disable/enable tracking pixel

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      |  NO
| New feature?  |  YES
| BC breaks?    |  NO
| Deprecations? |  NO
| Fixed issues  |  NO

## Description

As we all know, tracking open email action wia tracking image pixel is not a 100% accurate, well it is not even 50% accurate... but what it does in 100% of cases is "Images are not displayed. Display images below" popup even when you use #1512 ...

For two of my customers, it is NOT important to track open email action (due to its low accuracy) is important to track URL visits and show email to client with all images withnout ANY dialogs like "Images are not displayed. Display images below", its more about style for them so they wish to disable appending of tracking pixel in mautic and i fully understand their request. 

So this PR adds option to disable appending of tracking pixel image into email body, option can be found in Config -> Email Settings -> Message Settings: 
![tracking](https://cloud.githubusercontent.com/assets/1226241/14732318/dcd4d8e0-0859-11e6-9f2f-e49f303da6ad.png)

Default state is ENABLED (image is appended to email body as usually)

## Steps to test this PR

1. Clear cache
2. Create an email.
3. Send the email. The tracking image should show up in the received email body like this:
```html
<img style=3D"display: none;" height=3D"1" width=3D"1" src=3D"http:/=
/mautic.io/mautic-github/index_dev.php/email/5718bddb1f086.gif" alt=3D"Maut=
ic is open source marketing automation" />
```

4. Configure Mautic to send emails withnout tracking pixel image and send the email again.
There should be no tracking pixel image in email body.
